packages/babel-plugin-apply-mdx-type-props/package.json

```
@@ -25,11 +25,8 @@
     "remark",
     "babel"
   ],
-  "peerDependencies": {
-    "@babel/core": "^7.10.5"
-  },
   "dependencies": {
-    "@babel/helper-plugin-utils": "7.10.4"
+    "@babel/helper-plugin-utils": "^7.10.4"
```

would it make sense to broaden these to
```suggestion
    "@babel/helper-plugin-utils": "^7.0.0"
```
?

---

I’d say maybe later, I’m not too familiar with the internals there to know if we’re depending on internal features released in those minor versions